### PR TITLE
Mark `from` parameter as optional for `/messages`

### PR DESCRIPTION
**Pull request by jplatte**
Continuation of https://github.com/matrix-org/matrix-spec-proposals/pull/3638 on the new spec repo.

Preview: https://pr1002--matrix-spec-previews.netlify.app

### CODE REVIEW

File: data/api/client-server/message_pagination.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ paths:
         - in: query
           type: string
           name: from
+          x-changedInMatrixVersion: "1.3"
```
Comment (richvdh) [inline data/api/client-server/message_pagination.yaml:52]: this doesn't seem to be showing anything in the rendered spec.

Generally we need to describe *how* it changed, not just that it changed.

Comment (jplatte) [reply data/api/client-server/message_pagination.yaml:52]: So just prepend `**Since v1.3:**` to the new paragraph in the description, or something else?

Comment (richvdh) [reply data/api/client-server/message_pagination.yaml:52]: have a look at `data/api/client-server/definitions/client_event_without_room_id.yaml` for an example of how to use `x-changedInMatrixVersion` correctly

Comment (richvdh) [reply data/api/client-server/message_pagination.yaml]: hrm, it still doesn't seem to be rendering. We'll have to dig into what's wrong with the templates :/

Comment (richvdh) [reply data/api/client-server/message_pagination.yaml]: #1011 should fix this.

**Review by richvdh — APPROVED** [review]
lgtm

**Pull request merged**